### PR TITLE
SNOR-17: new sign in page

### PR DESCRIPTION
**Pull request by dyduch**

### CODE REVIEW

File: src/main/webapp/app/account/login/login.component.ts [hunk 1]
```diff
@@ -1,46 +1,39 @@
...
-    cancel() {
-        this.credentials = {
-            username: null,
-            password: null,
-            rememberMe: true
-        };
-        this.authenticationError = false;
-        this.activeModal.dismiss('cancel');
+    ngOnInit(): void {
+        this.success = false;
```
Comment (MarcinAman) [inline src/main/webapp/app/account/login/login.component.ts]: Why do we need this variable. I guess it is checking whether loging was successful but can't we just judge it from `authenticationError`?

File: src/main/webapp/app/layouts/navbar/navbar.component.html [hunk 2]
```diff
@@ -136,9 +136,9 @@
                         </a>
                     </li>
                     <li *ngSwitchCase="false">
-                        <a class="dropdown-item" (click)="login()" id="login">
+                        <a class="dropdown-item" routerLink="login" routerLinkActive="active" (click)="collapseNavbar()">
                             <fa-icon icon="sign-in-alt" fixedWidth="true"></fa-icon>
-                            <span jhiTranslate="global.menu.account.login">Sign in</span>
+                            <span jhiTranslate="global.menu.account.login">New Sign In</span>
```
Comment (MarcinAman) [inline src/main/webapp/app/layouts/navbar/navbar.component.html]: Why `New Sign In`?

**Review by MarcinAman — CHANGES_REQUESTED** [review]
Mostly ok, but please set the title on page with an endpoint: `http://<ip address>/#/admin/user-management/new`.

Also when i am creating a new user there is a `Activated` button. Can we remove it since it is unclicable?

File: src/main/webapp/app/layouts/navbar/navbar.component.html (hunk 2 above)
Comment (dyduch) [reply src/main/webapp/app/layouts/navbar/navbar.component.html]: forgot to change it back, doing it now

File: src/main/webapp/app/account/login/login.component.ts (hunk 1 above)
Comment (dyduch) [reply src/main/webapp/app/account/login/login.component.ts]: we don't need it at all now

**Comment by dyduch** [comment]
> 
> 
> Mostly ok, but please set the title on page with an endpoint: `http://<ip address>/#/admin/user-management/new`.
> 
> Also when i am creating a new user there is a `Activated` button. Can we remove it since it is unclicable?

I believe those two things are part of SNOR-3

**Comment by MarcinAman** [comment]
image

**Review by MarcinAman — APPROVED** [review]

**Pull request merged**